Explorer/Assets/DCL/Infrastructure/ECS/Unity/SceneBoundsChecker/CheckColliderBoundsSystem.cs

```diff
@@ -29,27 +28,23 @@ public partial class CheckColliderBoundsSystem : BaseUnityLoopSystem
 
         private readonly IPartitionComponent scenePartition;
         private readonly ParcelMathHelper.SceneGeometry sceneGeometry;
-        private readonly IPhysicsTickProvider physicsTickEntity;
         private Bounds auxiliaryBounds = new Bounds();
 
         /// <summary>
         ///     Throttle scheduling between fixed updates
         /// </summary>
         private int lastFixedFrameChecked;
 
-        internal CheckColliderBoundsSystem(World world, IPartitionComponent scenePartition, ParcelMathHelper.SceneGeometry sceneGeometry, IPhysicsTickProvider physicsTickEntity) : base(world)
+        internal CheckColliderBoundsSystem(World world, IPartitionComponent scenePartition, ParcelMathHelper.SceneGeometry sceneGeometry) : base(world)
         {
             this.scenePartition = scenePartition;
             this.sceneGeometry = sceneGeometry;
-            this.physicsTickEntity = physicsTickEntity;
         }
 
         protected override void Update(float t)
         {
-            int tick = physicsTickEntity.Tick;
-            if (tick == lastFixedFrameChecked) return;
```

Comment:
Can you move a tick check into `CheckGltfAsset` and execute that logic only if the tick has passed on `pbComponent` is dirty?

Reply:
In the end I could keep the original tick skipping optimization: https://github.com/decentraland/unity-explorer/pull/3748/commits/8f862948594b1a68999e40e56b631a4614cd2320